docs/index.html

```diff
@@ -195,7 +195,7 @@ <h4 class='speakers__title'>Speakers:</h4>
 </li>
 <li class='speakers__item'>
 <a class='speakers__link' href='https://twitter.com/jmcharnes' target='_blank'>
-<img alt='Jason Charnes' src='./assets/images/speakers/Jason-Charnes.jpg'>
+<img alt='Jason Charnes' src='./assets/images/speakers/Jason-charnes.jpg'>
```

Comment:
Hmm idk, might as well just change them all to be lowercase?

![2017-04-25 at 10 41 am](https://cloud.githubusercontent.com/assets/17035/25394516/d5e493c4-29a3-11e7-9334-57d4d65f9e92.jpg)